tools/msstatstmt/msstatstmt.xml

```diff
@@ -0,0 +1,379 @@
+            <param name="summaryPlot" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="Draw profile plots with protein summarization for each channel and MS run"/>
+        </section>
+    </inputs>
+    <outputs>
+        <data name="out_msstats_log" format="txt" label="${tool.name} on ${on_string}: MSstats log">
+            <filter>'msstats_log' in selected_outputs</filter>
+        </data>
+        <data name="out_msstatstmt_log" format="txt" label="${tool.name} on ${on_string}: MSstatsTMT log">
+            <filter>'msstatstmt_log' in selected_outputs</filter>
+        </data>
+        <data name="out_r_script" format="txt" label="${tool.name} on ${on_string}: Rscript">
+            <filter>'r_script' in selected_outputs</filter>
+        </data>
+        <data name="out_profile_plot" from_work_dir="ProfilePlot.pdf" format="pdf" label="${tool.name} on ${on_string}: Profile Plot">
+            <filter>'ProfilePlot' in selected_outputs</filter>
+        </data>
+        <data name="out_qc_plot" from_work_dir="QCPlot.pdf" format="pdf" label="${tool.name} on ${on_string}: QC Plot">
+            <filter>'QCPlot' in selected_outputs</filter>
+        </data>
+        <data name="out_group_comp" from_work_dir="ComparisonResult.tsv" format="tsv" label="${tool.name} on ${on_string}: Group Comparison">
+            <filter>group['group_comparison'] == 'true'</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="5">
+            <conditional name="input">
+                <param name="input_src" value="MSstats"/>
+                <param name="msstats_input" ftype="tabular" value="input.msstats.txt"/>
+            </conditional>
+            <param name="selected_outputs" value="msstats_log,msstatstmt_log,r_script,ProfilePlot,QCPlot"/>
+            <output name="out_msstats_log">
+                <assert_contents>
+                    <has_text text="1 level of Isotope type labeling in this experiment" />
+                </assert_contents>
+            </output>
+            <output name="out_msstatstmt_log">
+                <assert_contents>
+                    <has_text text="MSstatsTMT - proteinSummarization function" />
+                </assert_contents>
+            </output>
+            <output name="out_r_script">
+                <assert_contents>
+                    <has_n_lines n="53" />
+                </assert_contents>
+            </output>
+            <output name="out_profile_plot" file="ProfilePlot.pdf" compare="sim_size"/>
+            <output name="out_qc_plot" file="QCPlot.pdf" compare="sim_size"/>
+        </test>
+        <test expect_num_outputs="2">
+            <conditional name="input">
+                <param name="input_src" value="MaxQuant"/>
+                <param name="evidence" ftype="tabular" value="evidence.txt"/>
+                <param name="annotation" ftype="tabular" value="annotation.txt"/>
```

Comment:
better order would be evidence, proteingroups, annotation (msstats labelfree tool will also be changed to that order)